package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Webstudio Designer UI as a service",
   "scripts": {
     "build": "remix build",
-    "dev": "yarn setup && node -r dotenv/config node_modules/.bin/remix dev",
+    "dev": "yarn setup && HOST=localhost node -r dotenv/config node_modules/.bin/remix dev",
```

Comment:
what's required HOST?

Reply:
that was a experiemnt I forgot to remove